timeless/reservations/controllers.py

```diff
@@ -38,3 +38,14 @@ def delete_reservation_settings(self, reservation_id):
         raise Exception(
             "SettingsController#delete_reservation_settings not yet implemented"
         )
+
+
+class CommentsController:
+
```

Comment:
@Vatavuk Actually I do not understand why we need controllers for CRUD operations. For me it seems like controllers for the sake of controllers. I'm looking at `get_comment` and see 5 lines of code, instead of only one using `Comment.query.one`, which returns instance or raises exception (https://kite.com/python/docs/sqlalchemy.orm.query.Query.one). Flask even provides generic snippet for views: http://flask.pocoo.org/snippets/39/ . All these logic should be in generic view.. or generic controller which will fit to all CRUD views. I guess next time all guys just will copy-paste it and we will get mess. @emilianodellacasa please clarify is it fine or not

Reply:
@vryazanov I don't like this controller approach too. I've put it here just to be consistent. On the other hand I prefer to decouple views from the implementation of a particular use case, but for this tiny CRUD operations I agree let's leave the logic in the views, I will change it.

Reply:
@vladarefiev @emilianodellacasa Regarding generics, I agree, I will create a small generic view (just with retrieval operation), maybe than would be more clear how this would fit.